build.gradle.kts

```diff
@@ -50,21 +49,6 @@ allprojects {
         disabledRules by setOf("indent", "max-line-length")
     }
 
-    tasks.withType<KotlinCompile>().configureEach {
```

Comment:
Currently this is in `allProjects` so it also configures the overall project and its subprojects to use Java 8. Moving it to `setupBaseModule` makes it so it's only configured for Android subprojects.

Reply:
`KotlinOptions` now should be used in subprojects only, root project use the default Java (currently it was `Java11` embedded in Android Studio)

Reply:
@Goooler Do you have a source on that? I thought the JVM that runs the code is different from the language level target set by the `KotlinCompile` task.

Reply:
Not yet. 
You can change level to Java11 and compile for a test, it will be failed.
And I think arguments in `kotlinOptions` that you added are useless for root project.